thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs

```diff
@@ -29,6 +29,9 @@
         <label for="select-metric" class="control-label te-label te-label--taller required">Search for a metric, or
           {{#link-to "self-serve.import-metric" class="thirdeye-link-secondary thirdeye-link-secondary--inside"}}
             Import a Metric From InGraphs
+          {{/link-to}} or
+          {{#link-to "self-serve.import-presto-metric" class="thirdeye-link-secondary thirdeye-link-secondary--inside"}}
+            Import a Metric From Presto
```

Comment:
I am not sure if we should surface this yet! This will make the link visible in production. Maybe comment this code out for now and we can enable it for the ThirdEye users once we test it internally and are happy with the latency requirements.

Also make sure the public facing links are suppressed elsewhere in this PR if any.